packages/server/billing/helpers/removeTeamsLimitObjects.ts

```diff
@@ -3,6 +3,9 @@ import updateNotification from '../../graphql/public/mutations/helpers/updateNot
 import getKysely from '../../postgres/getKysely'
 
 const removeTeamsLimitObjects = async (orgId: string, dataLoader: DataLoaderWorker) => {
+  const operationId = dataLoader.share()
+  const subOptions = {operationId}
```

Comment:
A dataloader has a cache TTL of 30s if it's shared, 0 otherwise. If there is no other call which shares it before the mutation returns, it could already be pruned from the cache before the `share()` call happens.